bin/create_table.py

```diff
@@ -42,10 +45,32 @@ def extract_busco(file_in, file_out):
         print("busco_lineage", data["lineage_dataset"]["name"], sep=",", file=fout)
         print("busco_summary", '"' + data["results"]["one_line_summary"] + '"', sep=",", file=fout)
 
+def extract_qv(file_in, file_out):
+    out_dir = os.path.dirname(file_out)
+    make_dir(out_dir)
+
+    with open(file_in, "r") as fin, open(file_out, "a") as fout:
+        data = csv.DictReader(fin, delimiter="\t")
+        for row in data:
+            del row["Assembly"]
+            print("MERQURYFK_QV", row, sep=",", file=fout)
```

Comment:
To perhaps confirm with Karen, but I'm expecting we only need the QV value itself
```suggestion
            print("MERQURYFK_QV", row["QV"], sep=",", file=fout)
```